src/fermilib/utils/_plane_wave_hamiltonian.py

```diff
@@ -162,9 +163,7 @@ def plane_wave_hamiltonian(n_dimensions, grid_length, length_scale, geometry,
         if item[0] not in periodic_hash_table:
             raise ValueError("Invalid nuclear element.")
 
-    jellium_op = jellium_model(n_dimensions,
-                               grid_length,
-                               length_scale,
+    jellium_op = jellium_model(Grid(n_dimensions, grid_length, length_scale),
```

Comment:
To keep this PR small, other functions will be updated in follow-up PRs. I did one or two in this one so that it was clear how I was going to be using Grid.